app/src/main/java/de/westnordost/streetcomplete/quests/postbox_collection_times/AddPostboxCollectionTimes.kt

```diff
@@ -35,7 +35,21 @@ class AddPostboxCollectionTimes(o: OverpassMapDataDao) : SimpleOverpassQuestType
         // apparently mostly not in Latin America and in Arabic world and unknown in Africa
     )
 
-    override fun getTitle(tags: Map<String, String>) = R.string.quest_postboxCollectionTimes_title
+    override fun getTitleReplacements(tags: Map<String, String>, typeName: Lazy<String?>): Array<String?> {
+        for(nameTag in listOf("name", "brand", "operator")) {
+            if (tags.containsKey(nameTag)) {
+                val value = tags.getValue(nameTag)
+                return arrayOf(value)
+            }
+        }
+        return arrayOf()
+    }
```

Comment:
I suppose it is a matter of taste, but you really find this better to read than
```kotlin
override fun getTitleReplacements(tags: Map<String, String>, typeName: Lazy<String?>) = 
    arrayOf(tags["name"] ?: tags["brand"] ?: tags["operator"])
```
?

Reply:
What about case where neither `name` nor `brand` nor `operator` tag is present? Is it OK to return `arrayOf(null)`? From looking at the code it should work (not tested), but it seems a bit ugly to me.

Though this function is really verbose and it would be nice to shorten it.

Reply:
> Is it OK to return `arrayOf(null)`?

Yes